src/main/java/com/algorand/algosdk/transaction/MethodCallOption.java

```diff
@@ -0,0 +1,135 @@
+package com.algorand.algosdk.transaction;
+
+import com.algorand.algosdk.abi.Method;
+import com.algorand.algosdk.algod.client.model.TransactionParams;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MethodCallOption {
```

Comment:
nit: `MethodCallParams` might be a better name?

Reply:
Yes, I agree